mam/mam/message.h

```diff
@@ -54,6 +54,7 @@ typedef struct mam_msg_send_context_s {
   mam_spongos_t spongos;
   trint18_t ord;
   mam_mss_t *mss;
+  trit_t mss_root[MAM_MSS_PK_SIZE];
```

Comment:
In https://github.com/iotaledger/entangled/pull/911 the root is now serialized with the MSS so no need for this field